base/markdown/render/html.jl

```diff
@@ -56,6 +56,21 @@ function html(io::IO, md::List)
     end
 end
 
+function html(io::IO, md::Table)
+    withtag(io, :table) do
+        for (i, row) in enumerate(md.rows)
+            withtag(io, :tr) do
+                for c in md.rows[i]
+                    t = i == 1 ? :th : :td
```

Comment:
Very minor thing, but I find this a little hard to read without brackets around the ternary operator.